repo/incubating/mysql/versions/0/mysql-frameworkversion.yaml

```diff
@@ -0,0 +1,249 @@
+apiVersion: kudo.k8s.io/v1alpha1
+kind: FrameworkVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: mysql-57
+  namespace: default
+spec:
+  serviceSpec:
+  version: "5.7"
+  connectionString: ""
+  framework:
+    name: mysql
+    kind: Framework
+  parameters:
+  - name: BACKUP_FILE
+    description: "Filename to save the backups to"
+    default: "backup.sql"
+    displayName: "BackupFile"
+    trigger: backup
+  - name: PASSWORD
+    default: "password"
+  tasks:
+    deploy:
+      resources:
+      - mysql.yaml
+    init:
+      resources:
+      - init.yaml
+    pv:
+      resources:
+      - backup-pv.yaml
+    backup:
+      resources:
+      - backup.yaml
+    restore:
+      resources:
+      - restore.yaml
+    load-data:
+      resources:
+      - job.yaml
+    clear-data:
+      resources:
+      - job.yaml
+    query:
+      resources:
+      - job.yaml
+  templates:
+    mysql.yaml: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: mysql
+      spec:
+        ports:
+        - port: 3306
+        selector:
+          app: mysql
+        clusterIP: None
+      ---
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: mysql-pv
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+      ---
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: mysql
+      spec:
+        selector:
+          matchLabels:
+            app: mysql
+        strategy:
+          type: Recreate
+        template:
+          metadata:
+            labels:
+              app: mysql
+          spec:
+            containers:
+            - image: mysql:5.7
+              name: mysql
+              env:
+                # Use secret in real usage
+              - name: MYSQL_ROOT_PASSWORD
+                value: {{PASSWORD}}
+              - name: MYSQL_DATABASE
+                value: kudo
+              ports:
+              - containerPort: 3306
+                name: mysql
+              volumeMounts:
+              - name: mysql-persistent-storage
+                mountPath: /var/lib/mysql
+              livenessProbe:
+                exec:
+                  command: ["mysqladmin", "-p{{PASSWORD}}", "ping"]
+                initialDelaySeconds: 30
+                periodSeconds: 10
+                timeoutSeconds: 5
+              readinessProbe:
+                exec:
+                  # Check we can execute queries over TCP (skip-networking is off).
+                  command: ["mysql", "-p{{PASSWORD}}", "-h", "127.0.0.1", "-e", "SELECT 1"]
+                initialDelaySeconds: 5
```

Comment:
I am getting this in the beginning, not sure if we should increase the `initialDelaySeconds` ?

```
Events:
  Type     Reason            Age    From               Message
  ----     ------            ----   ----               -------
  Warning  FailedScheduling  2m20s  default-scheduler  persistentvolumeclaim "mysql-mysql-pv" not found
  Normal   Scheduled         2m19s  default-scheduler  Successfully assigned default/mysql-mysql-5ff986b94c-tqbsx to minikube
  Normal   Pulling           2m18s  kubelet, minikube  Pulling image "mysql:5.7"
  Normal   Pulled            2m7s   kubelet, minikube  Successfully pulled image "mysql:5.7"
  Normal   Created           2m7s   kubelet, minikube  Created container mysql
  Normal   Started           2m7s   kubelet, minikube  Started container mysql
  Warning  Unhealthy         2m1s   kubelet, minikube  Readiness probe failed: mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 2003 (HY000): Can't connect to MySQL server on '127.0.0.1' (111)
```

ultimately everything comes up healthy though.